### ISSUE TITLE
System Error when running Ganache 2.0.0 on linux

### ISSUE THREAD

**Issue by sowmyakannan** [trufflesuite/ganache#1237]
PLATFORM: linux
GANACHE VERSION: 2.0.0

EXCEPTION:
```
TypeError: Cannot read property 'nodes' of undefined
    at getStateVariables (/node_modules/truffle-decoder/dist/allocate/storage.js:146:24)
    at vars.concat.linearizedBaseContractsFromBase.map (/node_modules/truffle-decoder/dist/allocate/storage.js:154:73)
    at Array.map (native)
    at allocateContract (/node_modules/truffle-decoder/dist/allocate/storage.js:154:61)
    at Object.getStorageAllocations (/node_modules/truffle-decoder/dist/allocate/storage.js:28:23)
    at TruffleContractDecoder.init (/node_modules/truffle-decoder/dist/interface/contract-decoder.js:87:43)
    at Object.getDecodedTransaction (/src/truffle-integration/decode.js:60:11)
    at process.<anonymous> (/src/truffle-integration/index.js:108:31)
    at emitTwo (events.js:125:13)
    at process.emit (events.js:213:7)
    at emit (internal/child_process.js:768:12)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

APPLICATION LOG:
```
T+1574231ms: eth_getBlockByNumber
T+1574329ms: eth_getBlockByNumber
T+1574434ms: eth_estimateGas
T+1574523ms: eth_getBlockByNumber
T+1574684ms: eth_blockNumber
T+1574775ms: eth_sendTransaction
T+1574882ms:   Transaction: 0xe43c6ce55289eeddac4cab2baf70d83115ee2aab2d7804a38d5d492794535142
T+1574882ms:   Contract created: 0x0cf81ebd9153227e0d410282fa09372f5bf0c5c3
T+1574882ms:   Gas usage: 2210968
T+1574882ms:   Block Number: 1348
T+1574982ms:   Block Time: Thu Apr 11 2019 14:56:57 GMT+0400 (+04)
T+1575101ms: eth_getBlockByNumber
T+1575241ms: eth_getTransactionReceipt
T+1575370ms: eth_getCode
T+1575492ms: eth_getBlockByNumber
T+1575619ms: eth_getBlockByNumber
T+1575745ms: eth_sendTransaction
T+1575877ms:   Transaction: 0x45cd69f3819a28856d4c8fa4214616e13997d321fb95a004fb38132cb9f2b27c
T+1575877ms:   Gas usage: 26929
T+1575877ms:   Block Number: 1349
T+1575877ms:   Block Time: Thu Apr 11 2019 14:56:57 GMT+0400 (+04)
T+1576007ms: eth_getBlockByNumber
T+1576120ms: eth_getTransactionReceipt
T+1576218ms: evm_snapshot
T+1576327ms: eth_getBlockByNumber
T+1576427ms: eth_getBlockByNumber
T+1576531ms: eth_getBlockByNumber
T+1576638ms: eth_getBlockByNumber
T+1576738ms: eth_getBlockByNumber
T+1576846ms: eth_getBlockByNumber
T+1576943ms: eth_blockNumber
T+1577043ms: eth_getBlockByNumber
T+1577138ms: eth_getBlockByNumber
T+1577242ms: eth_sendTransaction
T+1577357ms:   Transaction: 0x13b7cc63137d43f94b07c1b996a5cab150473cc7ca4623c99152a873d28401c9
T+1577357ms:   Gas usage: 45837
T+1577357ms:   Block Number: 1350
T+1577357ms:   Block Time: Thu Apr 11 2019 14:56:58 GMT+0400 (+04)
T+1577448ms: eth_getBlockByNumber
T+1577707ms: eth_getTransactionReceipt
T+1577707ms: eth_getBlockByNumber
T+1577804ms: eth_call
T+1577906ms: eth_getBlockByNumber
T+1578010ms: eth_sendTransaction
T+1578119ms:   Transaction: 0xf839fc61138e5b6f10718e172563dcc8de7833594e29896d241ff6f249653cd6
T+1578119ms:   Contract created: 0x7b6cace94ee0565785927de6b876f925f73f153f
T+1578119ms:   Gas usage: 2091577
T+1578119ms:   Block Number: 1351
T+1578119ms:   Block Time: Thu Apr 11 2019 14:56:58 GMT+0400 (+04)
T+1578222ms: eth_getBlockByNumber
T+1578521ms: eth_getTransactionReceipt
T+1578521ms: eth_getCode
T+1578521ms: eth_getBlockByNumber
T+1578521ms: eth_getBlockByNumber
T+7032275ms: web3_clientVersion
T+7032467ms: eth_getBalance
T+7032746ms: eth_getCode
T+7032875ms: eth_getCode
T+7032998ms: eth_getCode
T+7033124ms: eth_getCode
T+7033233ms: eth_gasPrice
T+7033349ms: eth_gasPrice
T+7033493ms: eth_call
T+7033582ms: eth_call
T+7033681ms: eth_getTransactionCount
T+7033781ms: eth_sendRawTransaction
T+7041695ms:   Transaction: 0xcbc30aa594967ddb45d4d0fb6eaa7ef8983721c0a82db72d99b72b05eed80146
T+7041695ms:   Gas usage: 28577
T+7041695ms:   Block Number: 1352
T+7041811ms:   Block Time: Thu Apr 11 2019 16:31:05 GMT+0400 (+04)
T+7041811ms:   Runtime Error: revert
T+7041910ms: eth_getBlockByNumber
T+8083463ms: eth_getBlockByNumber
T+8083631ms: eth_sendTransaction
T+8091885ms:   Transaction: 0x4fe96528a26d49acb14f85c0147559e7693ae250e740fd543c8295b406cb0f1b
T+8091885ms:   Gas usage: 45773
T+8091885ms:   Block Number: 1353
T+8091885ms:   Block Time: Thu Apr 11 2019 16:48:36 GMT+0400 (+04)
T+8091982ms: eth_getBlockByNumber
T+8092434ms: eth_getTransactionReceipt
T+8309355ms: eth_getBlockByNumber
T+8335075ms: eth_getBlockByNumber
T+8335235ms: eth_call
T+8419334ms: eth_getBlockByNumber
T+8419505ms: eth_call
T+8427301ms: eth_getBlockByNumber
T+8427456ms: eth_call
T+8823350ms: eth_getBlockByNumber
T+8823503ms: eth_call
T+8932564ms: web3_clientVersion
T+8932869ms: eth_getBalance
T+8933069ms: eth_getCode
T+8933333ms: eth_getCode
T+8933580ms: eth_getCode
T+8933847ms: eth_getCode
T+8934043ms: eth_gasPrice
T+8934189ms: eth_gasPrice
T+8934289ms: eth_call
T+8934400ms: eth_call
T+8934512ms: eth_getTransactionCount
T+8934660ms: eth_sendRawTransaction
T+8942863ms:   Transaction: 0xf3c44eac53869adf7b3d55180e149169a869809a4011771aafb5986ce609f513
T+8942863ms:   Gas usage: 28577
T+8942863ms:   Block Number: 1354
T+8942964ms:   Block Time: Thu Apr 11 2019 17:02:46 GMT+0400 (+04)
T+8942964ms:   Runtime Error: revert
T+8943082ms: eth_getBlockByNumber
T+9509147ms: eth_getBlockByNumber
T+9509330ms: eth_getBlockByNumber
T+9509534ms: eth_call
T+9696022ms: eth_getBlockByNumber
T+9696205ms: eth_call
T+10080470ms: eth_unsubscribe
T+10080598ms: eth_unsubscribe
T+10080736ms: eth_unsubscribe
T+10080918ms: eth_unsubscribe
T+10081056ms: eth_unsubscribe
```

**Comment by davidmurdoch** [comment]
Hey @sowmyakannan, thanks for opening this issue. This bug should now be fixed in our latest beta (https://github.com/trufflesuite/ganache/releases/tag/v2.0.1-beta.0). Feel free to try out the beta now if you can't wait for the fix to land in v2.0.1 (in about a week from now).

**Comment by davidmurdoch** [comment]
_Now_ this issue should be fixed in the latest beta! https://github.com/trufflesuite/ganache/releases/tag/v2.0.2-beta.0
